packages/server/src/tests/amp/ampEpicSelection.ts

```diff
@@ -45,6 +45,7 @@ export const getAmpExperimentData = async (tests: AmpEpicTest[]): Promise<AmpExp
     return ampExperiments;
 };
 
+// TODO: work out how to add prices object to AMP Epics
```

Comment:
there's currently no requirement for these templates in the epic, so I don't think we should consider this until it's needed. The tool validates any templates to ensure they're supported by the channel/platform